apps/tilbakemelding-api/src/main/java/no/nav/registre/testnorge/tilbakemeldingapi/consumer/ProfilApiConsumer.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.registre.testnorge.tilbakemeldingapi.consumer;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.registre.testnorge.libs.dto.profil.v1.ProfilDTO;
+import no.nav.registre.testnorge.libs.oauth2.domain.AccessScopes;
+import no.nav.registre.testnorge.libs.oauth2.domain.AccessToken;
+import no.nav.registre.testnorge.libs.oauth2.domain.ClientCredential;
+import no.nav.registre.testnorge.libs.oauth2.service.OnBehalfOfGenerateAccessTokenService;
+import no.nav.registre.testnorge.tilbakemeldingapi.consumer.credentials.TilbakemeldingApiClientCredential;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Slf4j
+@Component
+
```

Comment:
Legg til @DependencyOn("profil-api") under @Component. Glemte det i går. Det er for å oppdatere oversikten over hvilke apper som er avhengig av hverandre (https://testnorge-avhengighetsanalyse-frontend.dev.adeo.no/)